lib/ButtonNew/ButtonAvatar/ButtonAvatar.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { string, bool } from 'prop-types';
+import { Avatar } from 'lib';
+import cx from 'classnames';
+import { ButtonBase } from '../ButtonBase';
+
+export function ButtonAvatar({
+  initials,
+  url,
```

Comment:
💚 This comment is very green, but coming to this component for the first time I had no idea what `url` was for until I dipped into `<Avatar />` - maybe `avatarImageUrl` or something similar would be more descriptive!